src/js/components/Select/SelectContainer.js

```diff
@@ -398,6 +402,7 @@ const SelectContainer = forwardRef(
                         </Text>
                       </OptionBox>
                     );
+
                   // if we have a child, turn on plain, and hoverIndicator
 
                   return (
```

Comment:
Would it make any sense to add `selected` as a prop to `Button` and then do that styling, in the same we do `active` styling? I'm not sure, so just wondering.

Reply:
If the styling for a selected option is done in Button it changes the behavior when a selected option is hovered over the background color changes to the `active` color. Current behavior for Select is that when a selected option is hovered over the background color doesn't change.

Reply:
I'm wondering if we should reconsider that Button behavior. Is there anywhere else where we use `<Button selected />`? I noticed that there is currently no `selected` property documented for Button. I was wondering if we should add it, and have its behavior be what you mention, that hover doesn't affect its styling.

Reply:
I didn't find other places where `<Button selected />` is used. The places where I thought it might come up (like on Calendar) are not doing the selected styling on the button, similar to how Select is currently set up.

It makes sense to me to change the selected behavior on button so that the hover isn't changing the background. I can go ahead and do that and add the selected prop to the documentation.

Reply:
OK. I think we might consider not adding it to the documentation to start with, and just consider this an internal usage for now. Then, if demand arises, we can add it. I say this in the spirit of not growing our published API surface unless there are concrete needs for it.